api/pkg/db/model/model.go

```diff
@@ -21,19 +21,19 @@ type (
 
 	Catalog struct {
 		gorm.Model
-		Name       string
-		Type       string
-		URL        string
-		Owner      string
+		Name       string `gorm:"size:100;not null;unique"`
+		Type       string `gorm:"size:100;not null;unique"`
+		URL        string `gorm:"size:100;not null;unique"`
+		Owner      string `gorm:"size:100;not null;unique"`
```

Comment:
Do we need this field? Can we remove this until a feature requires it? I am also thinking that the mapping may be one to many; i.e. a catalog is owned by multiple owners ..